about.md

```diff
@@ -1,6 +1,14 @@
 ---
 layout: page
-title: About me 
+title: My name is Joffrey and I <3 development 
 ---
 
-This is a static page. It could be an 'about page' if you'd like.
+My name is Joffrey Kern, I'm living in Strasbourg, France and working for Euro Information Developement on a project called LyfPay as .NET Architect. I'm also co-founder of the [Microsoft User Group Strasbourg](https://www.meetup.com/MugStrasbourg/). I'm a [developer](./projects.html) before anything else and I like to share my knowledge with this [blog](./blog) or [conferences](./speaker.html).
+
+The goal of this blog is to motivate myself on working on new technologies and share my opinions on it. I'm blogging with the language of Shakespeare to improve my English, this is not my mother tongue ! About the content, if you found a bad sentence, grammatical fault or any thing is not "good English", feel [free to tell me what is wrong and why is wrong](https://github.com/joffreykern/joffreykern.github.io/issues) ! My blog is opensource and everyone is welcome to improve the content of it !
```

Comment:
Je ne suis pas sure à 100% mais j'aurais mis "to motivate myself to work"

Et il te manque un mot "anything that is not good English".